source/bson/macros.txt

```diff
@@ -1,8 +1,8 @@
 .. _scala-macros:
 
```

Comment:
Q: should you also change the title in the TOC? "Macros" is pretty different than "Create a Codec to Convert Case Class Data" and it might be confusing to click on, maybe it could be "Case Class Codecs" in the toc

Reply:
Good idea, thanks!